frontend/src/components/VHeader/VHeaderMobile/VFilterTab.vue

```diff
@@ -1,14 +1,18 @@
 <template>
-  <VTab id="filters" size="medium" class="gap-x-2">
+  <VTab id="filters" size="medium" class="gap-x-2" :aria-label="ariaLabel">
```

Comment:
Kind of a nit, but in the interest of avoiding unnecessary lines of code, you can use `$tc` in template contexts and avoid needing to create the computed, call `useI18n`, add the `setup`, etc.

```diff
diff --git a/frontend/src/components/VHeader/VHeaderMobile/VFilterTab.vue b/frontend/src/components/VHeader/VHeaderMobile/VFilterTab.vue
index eb9bda1d7..7ae0385ee 100644
--- a/frontend/src/components/VHeader/VHeaderMobile/VFilterTab.vue
+++ b/frontend/src/components/VHeader/VHeaderMobile/VFilterTab.vue
@@ -1,13 +1,16 @@
 <template>
-  <VTab id="filters" size="medium" class="gap-x-2" :aria-label="ariaLabel">
+  <VTab
+    id="filters"
+    size="medium"
+    class="gap-x-2"
+    :aria-label="$tc('header.filters.withCount', appliedFilterCount)"
+  >
     <VFilterIconOrCounter :applied-filter-count="appliedFilterCount" />
     <h2 class="label-regular">{{ $t("filters.title") }}</h2>
   </VTab>
 </template>
 <script lang="ts">
-import { computed, defineComponent } from "vue"
-
-import { useI18n } from "~/composables/use-i18n"
+import { defineComponent } from "vue"
 
 import VFilterIconOrCounter from "~/components/VHeader/VFilterIconOrCounter.vue"
 import VTab from "~/components/VTabs/VTab.vue"
@@ -21,12 +24,5 @@ export default defineComponent({
       default: 0,
     },
   },
-  setup(props) {
-    const i18n = useI18n()
-    const ariaLabel = computed(() =>
-      i18n.tc("header.filters.withCount", props.appliedFilterCount)
-    )
-    return { ariaLabel }
-  },
 })
 </script>
```